test/ex_doc/autolink_test.exs

```diff
@@ -449,6 +449,32 @@ defmodule ExDoc.AutolinkTest do
     assert_unchanged(~m"`Unknown`")
 
     assert_unchanged(~m"[Blank](about:blank)")
+
+    # warnings with functions starting with an underscore
+    # TODO in a following commit: Split this test into tests inside a describe block
+    assert warn("WithModuleDoc._foo/1",
+             file: "lib/with_module_doc.ex",
+             id: "WithModuleDoc._foo/1"
```

Comment:
`file` and `id` don't seem relevant here any reason for including these?

Reply:
Removed.